scripts/release_ppa.sh

```diff
@@ -51,17 +54,43 @@ is_release() {
     [[ "${branch}" =~ ^v[0-9]+(\.[0-9]+)*$ ]]
 }
 
-keyid=379F4801D622CDCF
-email=builds@ethereum.org
+# source keyid and email from .release_ppa_auth
+if [ -e .release_ppa_auth ]
+then
+    # shellcheck source=/dev/null
+    source .release_ppa_auth
+fi
+
+if [ -z "$LAUNCHPAD_KEYID" ] || [ -z "$LAUNCHPAD_EMAIL" ]
+then
+    echo "Error: Couldn't find variables \$LAUNCHPAD_KEYID or \$LAUNCHPAD_EMAIL in sourced file .release_ppa_auth (check top comment in $0 for more information)."
+    exit 2
+fi
+
 packagename=solc
 
-static_build_distribution=impish
+# This needs to be a still active release
+static_build_distribution=focal
 
-DISTRIBUTIONS="focal impish jammy kinetic"
+DISTRIBUTIONS="focal jammy kinetic"
+
+checkDputEntries () {
+    if ! grep "${1}" /etc/dput.cf -q; then
+        echo "Error: Missing ${1} section in /etc/dput.cf (check top comment in ${0} for more information)."
```

Comment:
```suggestion
        >&2 echo "Error: Missing ${1} section in /etc/dput.cf (check top comment in ${0} for more information)."
```

Also, with `scripts/common.sh` these checks might be concise enough to skip the function completely:
```
grep "\[ethereum\]" /etc/dput.cf --quiet || \
    fail "Error: Missing [ethereum] section in /etc/dput.cf (check top comm1ent in ${0} for more information)."
```
There's also `assertFail` you could use instead of `fail` but I guess this is not appropriate here since not having this section is still user error, right?